plugins/09_Shadowlands/zones/zereth_mortis.lua

```diff
@@ -2046,3 +2046,17 @@ map.nodes[63206050] = Soulshape({
         Item({item = 189990, quest = 65518, covenant = NIGHTFAE}) -- Bee Soul
     }
 }) -- Bee Soul
+
+-------------------------------------------------------------------------------
+------------------------------- VE'NARI'S FATE --------------------------------
+-------------------------------------------------------------------------------
+
+map.nodes[47438855] = ns.node.Node({
```

Comment:
Change this to be a `Collectible` instance so that is does not appear on alts once the toy is collected.